fxa-auth-db-server/docs/DB_API.md

```diff
@@ -249,7 +250,7 @@ Parameters.
 
 Each token takes the following fields for it's create method respectively:
 
-* sessionToken : data, uid, createdAt
+* sessionToken : data, uid, createdAt, userAgent, os, deviceType
```

Comment:
> Okay, so we have: [...]

Looks good!

> Where ua-parser returns Other, we want to replace that with the empty string right?

Yes, or perhaps even an explicit NULL - whichever you think seems to fit best